examples/app/.bazelrc

```diff
@@ -0,0 +1,9 @@
+# Print test logs for failed tests
```

Comment:
we are accumulating a bunch of copy-paste code - what do you think about making these symlinks in the git repo?

Reply:
Sounds good. Separate PR?